.github/workflows/translation_keys.yml

```
@@ -0,0 +1,60 @@
+name: Translation keys
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    if: github.repository_owner == 'gristlabs'
    runs-on: ubuntu-latest
```

What do you think about disabling it on forks?